src/h3lib/include/h3Index.h

```diff
@@ -192,5 +186,6 @@ H3Index _h3RotatePent60ccw(H3Index h);
 H3Index _h3RotatePent60cw(H3Index h);
 H3Index _h3Rotate60ccw(H3Index h);
 H3Index _h3Rotate60cw(H3Index h);
+DECLSPEC H3Index _zero_index_digits(H3Index h, int start, int end);
```

Comment:
Why is this function using snake case?

Reply:
I guess I thought I could get away with it, as the functions above *involve* an underscore. 🙃